.github/.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+    "templateVersion": "0.1",
+    "product": {
+        "name": "Upward PHP",
+        "description": "An implementation of the UPWARD spec in PHP"
+    },
+    "contacts": {
+        "team": {
+            "name": "The Untitled Geese",
+            "DL": "Grp-Untitled-Geese",
+            "slackChannel": "pwa"
+        }
+    },
+    "ticketTracker": {
+        "functionalJiraQueue": {
+            "projectKey": "PWA",
+            "component": ""
+        },
+        "securityJiraQueue": {
+            "projectKey": "MAGREQ",
+            "component": "PWA"
+        }
+    },
+    "staticScan": {
+        "enable": false,
+        "frequency": "quarterly",
+        "customName": "UPWARD PHP Static Scan",
+        "branchesToScan": ["develop"]
```

Comment:
Should `branchesToScan` include "master" ? Looks like `master` is more up-to-date than `develop`